apps/meteor/server/models/raw/LivechatDepartment.ts

```diff
@@ -330,6 +336,14 @@ export class LivechatDepartmentRaw extends BaseRaw<ILivechatDepartment> implemen
 
 		return this.find(query, options);
 	}
+
+	findNotArchived(options: FindOptions<ILivechatDepartment> = {}): FindCursor<ILivechatDepartment> {
+		const query = {
+			$or: [{ archived: { $exists: false } }, { archived: false }],
```

Comment:
can `archived` have all the three states? like `false`, `true` and `undefined`? 

well maybe my bad on previous review, but ideally it should have only two states.. either: 
- always `true` or `false`
- or `true/false` and `undefined`

anyways, this query could be simpler, like `{ archived: { $ne: true } }` .. and unfortunately this query doesn't uses the most of the index.. negative conditions must go through all records anyways (the index will have only the records **with** the field, but the query also matches records that are not there.

Reply:
Sadly yes, a `never archived` department won't have the property `archived` in it. Only after a dep is archived then the property becomes `true`|`false`.

Good part is departments never grow that big for indexes to be critical, so both $ne: true & $exists: false should have similar perf.